example/ipsec_crypto/odp_ipsec_stream.c

```diff
@@ -65,6 +65,14 @@ void init_stream_db(void)
 	memset(stream_db, 0, sizeof(*stream_db));
 }
 
+void free_stream_db_intf(void)
+{
+	for (int i = 0; i < MAX_DB; i++) {
+		free((void *)(intptr_t)stream_db->array[i].input.intf);
```

Comment:
The casts are very ugly and should be avoided. I suppose you use them to get rid of the constness of the pointer, but I think it would be better to just remove the const specifier from the type.